src/cow-react/modules/swap/containers/EthFlowStepper/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { EthFlowStepper as Pure, SmartOrderStatus } from '@cow/modules/swap/pure/EthFlow/EthFlowStepper'
+import { useDetectNativeToken } from '@cow/modules/swap/hooks/useDetectNativeToken'
+import { Order } from 'state/orders/actions'
+import { NATIVE_CURRENCY_BUY_ADDRESS } from 'constants/index'
+
+type Props = {
+  storeOrder: Order | undefined
+}
+
+export function EthFlowStepper(props: Props) {
+  const { storeOrder } = props
+  const { native } = useDetectNativeToken()
+
+  const state = mapOrderToEthFlowStepperState(storeOrder)
+  const isEthFlowOrder = getIsEthFlowOrder(storeOrder)
+
+  if (!storeOrder || !state || !isEthFlowOrder) {
+    return null
+  }
+
+  const nativeTokenSymbol = native.symbol as string
+  const tokenLabel = storeOrder.outputToken.symbol as string
+
+  const order = {
```

Comment:
Please, set type to the const

Reply:
Let me know what you think of this
14fd1976bb68809145273489e697b3c2ed9f46b6

I thought against useMemo because there are no expensive operations to be memoized

Reply:
It's great! I love this approach